handlers/search.go

```diff
@@ -120,7 +120,7 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 			break
 		}
 		result = &types.SearchAheadBlocksResult{{
-			Block: block.Number,
+			Block: fmt.Sprintf("%v", block.Number),
```

Comment:
Changed this to string as the `Block` field is defined as `omitempty`. For an `uint64`, the value `0` is empty so for block 0 only the `Hash` reached the frontend.